debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleClobDataTypeIT.java

```diff
@@ -258,11 +258,11 @@ public void shouldStreamInlineClobDataTypeValues() throws Exception {
         record = records.recordsForTopic(topicName("CLOB_TEST")).get(0);
         VerifyRecord.isValidDelete(record, "ID", 1);
 
-        // clob fields will never have a "before" state
+        // clob fields will never have a "before" state; emitted with unavailable value placeholder
         Struct before = before(record);
         assertThat(before.get("ID")).isEqualTo(1);
-        assertThat(before.get("VAL_CLOB")).isNull();
-        assertThat(before.get("VAL_NCLOB")).isNull();
+        assertThat(before.get("VAL_CLOB")).isEqualTo(getUnavailableValuePlaceholder(config));
```

Comment:
That behavioral change is worth being called out in the change notes.

Reply:
Added label to the Jira issue and added a comment with the expected note.

Reply:
Thx!